content/kots-cli/admin-console/generate-manifests.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2021-07-29
+linktitle: "generate-manifests"
+title: kots admin-console generate-manifests
+weight: 90130
+---
+
+If you’d rather use kubectl or another workflow to deploy to a cluster, run `kots admin-console generate-manifests` to create a directory on the workstation containing the KOTS Admin Console manifests. This workflow enables additional customization of the Admin Console before deploying.
```

Comment:
This could maybe be reworked to align a bit with our contribution guidelines: 
Running this command will create a directory on the workstation containing the KOTS Admin Console manifests. These assets can be used to deploy KOTS to a cluster through other workflows, such as kubectl, to provide additional customization of the Admin Console before deploying.

Reply:
Ah, I was trying to figure out a good way to reword that first sentence to remove the "you" but didn't come up with anything. Thanks for the suggestion!